### PR TITLE
chore: setup readme file

### LINKED ISSUE #3: chore: setup Readme file

### PR DESCRIPTION
**Pull request by Paccyfic**
### What does this PR do?
This PR adds a structured README.md file for the Medica app project . The README includes a detailed description of the app, setup instructions, endpoints documentation, and contribution guidelines.

### Description :
-Created a README.md file .
-Added sections for project description, documentation, setup instructions, testing, contribution guidelines, and deployment.
-Included placeholders for badges, endpoints, dependencies, and testing instructions.
### Screenshot :
Screenshot 2024-05-01 at 11 55 14

 Review it and feel free to merge it to the main branch .Your feedback is appreciated!

### CODE REVIEW

**Review by mr3nz1 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
-# matadors-rn-medica
\ No newline at end of file
+# Medica App
+
+Url;
```
Comment (Nkbtemmy) [inline README.md]: This is not needed

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+
+### Getting Started
+
+1. Clone the repository: 
+git clone https://github.com/atlp-rwanda/matadors-rn-medica.git
+
+2. Navigate to the project directory:
+
+3. Install dependencies:
+npm install
```
Comment (Nkbtemmy) [inline README.md]: Highlight this command

File: README.md [hunk 3]
```diff
@@ -1 +1,107 @@
...
+
+4. Set up environment variables:
+- Create a `.env` file based on `.env.example` and fill in necessary values.
+
+### Run The Service
+
+1. Start the Expo development server:
+npm start
```
Comment (Nkbtemmy) [inline README.md:68]: Again this one

File: README.md [hunk 4]
```diff
@@ -1 +1,107 @@
...
+
+2. Use the Expo client on your mobile device or simulator to run the app.
+
+### Microservices
+
+None at the moment.
```
Comment (Nkbtemmy) [inline README.md]: This is not needed

File: README.md [hunk 5]
```diff
@@ -1 +1,107 @@
...
+
+## Testing
+
+To run tests, use the following command:
+npm test
```
Comment (Nkbtemmy) [inline README.md:76]: Also this one highlight it

File: README.md [hunk 6]
```diff
@@ -1 +1,107 @@
...
+
+## Contribute
+
+Contributions are welcome! To contribute to this repository, please follow these steps:
+1. Fork the repository.
+2. Create a new branch (`git checkout -b feature/my-feature`).
+3. Commit your changes (`git commit -am 'Add new feature'`).
+4. Push to the branch (`git push origin feature/my-feature`).
```
Comment (Nkbtemmy) [inline README.md:84]: Please make sure your branches are created as you said.

File: README.md [hunk 7]
```diff
@@ -1 +1,107 @@
...
+5. Create a new Pull Request.
+
+## Deployment
+
+The Medica app is deployed using [deployment platform]. To deploy updates:
+1. [Step 1 for deployment]
+2. [Step 2 for deployment]
+3. [Step 3 for deployment]
```
Comment (Nkbtemmy) [inline README.md]: Remove this ones

File: README.md [hunk 8]
```diff
@@ -1 +1,107 @@
...
+
+
+  ## Contact
+
+  Feel free to reach out to me on my email:
+  example@gmail.com  
+
+  ## License
+
+  None At the moment !!
```
Comment (Nkbtemmy) [inline README.md:96]: Better put your contributors list here including your manager and TTL

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]

File: README.md (hunk 8 above)
Comment (Paccyfic) [reply README.md:96]: you mean on the contacts ..??

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**